server/controller/userController.js

```diff
@@ -44,32 +48,42 @@ class User {
         message: 'Invalid user ID/phone number',
       });
     }
-    return pool.query(selQuery)
+    return pool.query(selQuery1)
       .then((data) => {
         if (data.rowCount === 1) {
           return res.status(409).send({
             status: 409,
             message: 'Email address has already been used',
           });
         }
-        return pool.query(insQuery)
+        return pool.query(selQuery2)
           .then((user) => {
-            if (user) {
-              const token = jwt.sign({
-                id: user.rows[0].id,
-                email: user.rows[0].email,
-                isAdmin: user.rows[0].is_admin,
-              }, process.env.SECRET_KEY, {
-                expiresIn: '24h',
-              });
-              res.status(201).json({
-                status: 201,
-
-                data: [token, user.rows[0].user_id, user.rows[0].first_name, user.rows[0].last_name,
-                  user.rows[0].other_name, user.rows[0].passport_url, user.rows[0].email,
-                  user.rows[0].is_admin],
+            if (user.rowCount === 1) {
+              return res.status(409).send({
+                status: 409,
+                message: 'User ID is invalid',
               });
             }
+            return pool.query(insQuery)
+              .then((userData) => {
+                if (userData) {
+                  const token = jwt.sign({
+                    id: user.rows[0].id,
+                    email: user.rows[0].email,
```

Comment:
Why do you have the user email as part of your token, confirm if and why you need to add email to you token